control/server.py

```diff
@@ -401,6 +401,18 @@ def _start_spdk(self, omap_state):
         for trtype in spdk_transports.split():
             self._create_transport(trtype.lower())
 
+        with self.ping_lock:
+            try:
+                ret_ver = spdk.rpc.spdk_get_version(self.spdk_rpc_ping_client)
+                try:
+                    ver_str = ret_ver["version"]
+                    self.logger.info(f"Started SPDK with version \"{ver_str}\"")
+                except KeyError:
+                    self.logger.error(f"Can't find SPDK version string in {ver_ret}")
+            except Exception:
+                self.logger.exception(f"Can't read SPDK version")
```

Comment:
wouldn't it better to add a print more info of the given exception with e/{e}
```
            except Exception as e:
                self.logger.exception(f"Unexpected error occurred while reading SPDK version: {e}")
```

Reply:
@barakda we already get the exception details from self.logger.exception(). We only need to add a title.